test/Image3.rb

```diff
@@ -1027,6 +1027,15 @@ def test_write
     img = Magick::Image.read('test.0')
     assert_equal('JPEG', img.first.format)
     FileUtils.rm('test.0')
+
+    if Magick.init_formats.has_key?("WEBP")
```

Comment:
Does it make sense to have the `else` case here? Does it throw an error in our code or does that come from ImageMagick?

Reply:
Make sense. Seems there is no problem if drop `else` case. I will remove `else`.

Reply:
Sorry, can we remove the `if`, too? I think ideally for our test suite we'd always run the code below.

Reply:
OK :)